spacy/tokenizer.pyx

```diff
@@ -458,9 +455,11 @@ cdef class Tokenizer:
         if string:
             if self._try_specials_and_cache(hash_string(string), tokens, has_special, with_special_cases):
                 pass
-            elif (self.token_match and self.token_match(string)) or \
-                    (self.url_match and \
-                    self.url_match(string)):
+            elif (
+                self.token_match and self.token_match(string) or
+                self.url_match and self.url_match(string)
```

Comment:
Does the linter actually want to remove the parentheses here? (I think it's technically correct but I strongly prefer keeping the parentheses for clarity.)

Reply:
No, I've taken that liberty. Readded in https://github.com/explosion/spaCy/pull/12694/commits/3583f94dab4ef46fd8322c93746fa88d061707df.

Reply:
Yea, I also strongly prefer keeping the parentheses for clarity in cases like this!